pyproject.toml

```diff
@@ -30,6 +30,7 @@ dynamic = ["version"]
 Documentation = "https://wheel.readthedocs.io/"
 Changelog = "https://wheel.readthedocs.io/en/stable/news.html"
 "Issue Tracker" = "https://github.com/pypa/wheel/issues"
+Repository = "https://github.com/pypa/wheel/"
```

Comment:
Is this more commonly called `Source`? It usually is in project that call `Homepage` the documentation, like https://pypi.org/project/pip/. Other projects that call documentation `Documentation` call this `Homepage`, like https://pypi.org/project/setuptools/.

Reply:
`Homepage` is annoyingly ambiguous, so let's at least not use that. As for `Repository` vs `Source`, I have no strong opinions one way or the other.

Reply:
Pip uses `Source` so I think we could go with that.

Reply:
Pip uses `Source` so I think we could go with that.

Reply:
Pip uses "Source".
```suggestion
Source = "https://github.com/pypa/wheel/"
```

Reply:
Pip uses "Source".
```suggestion
Source = "https://github.com/pypa/wheel/"
```